satpy/readers/vii_base_nc.py

```diff
@@ -84,7 +84,7 @@ def get_dataset(self, dataset_id, dataset_info):
                 variable = self._perform_interpolation(variable)
 
             # Perform the calibration if required
-            if dataset_info['calibration'] is not None:
+            if dataset_info.get('calibration', None) is not None:
```

Comment:
```suggestion
            if dataset_info.get('calibration') is not None:
```
`None` is already the default return value for `get`.
Also, should we issue a warning here if no calibration info is found?

Reply:
Already in the vii_l1b_nc we expect some datasets to not have calibration for example "observation_azimuth".